db/schema.sql

```diff
@@ -30,4 +30,26 @@ CREATE TABLE interest (
   interest varchar(255) NOT NULL,
   interest_desc varchar(255) NOT NULL,
   PRIMARY KEY (id)
+);
+
+--Create attendeeInt table.
+CREATE TABLE attendeeInt (
+  id int NOT NULL AUTO_INCREMENT,
+  attendee_name varchar(100) NOT NULL,
```

Comment:
instead of using attendee_name you need to use attendee_ID. As that is the primary key on the attendee table. 

Same with interest and event name below. 

Otherwise looks great!